### ISSUE TITLE
Config file is not being copied 

### ISSUE THREAD

**Issue by Jorge-Cov-Ap** [feedsbrain/instabot-docker#9]
Once the container is started, the config file is not being copied inside

I've edited the config file of this project and then I inspect the one that is inside of the container and they are different (inside of the container is the default one)

**Comment by feedsbrain** [comment]
The config folder should be mounted through volume: https://github.com/feedsbrain/instabot-docker/blob/master/docker-compose.yml#L15

**Comment by pleasemarkdarkly** [comment]
Hi, I am having the same problem. I updated the Docker to the latest version (versus the --ignore-update) and even changed the name of the configuration file. When I exec into the container it looks ok, even tried setting the environment variable INSTABOT_CONFIG_FILE but the logs only show that it wants to like dogs, cats and cars, which is mystifying. I'd be happy to share my files.

`2019-12-24 11:00:03,404 - instagram_user - InstaBot - DEBUG - Getting media by tag: cat`

Update: So I created a docker container instance from scratch on a completely different system with whatever...the point is that 'cat' is still showing up. So it's clearly not your issue. But it IS an issue. 

Update 2: It appears to be a problematic yml which is causing this behavior. When I took out some of the more exotic yml parameters, it worked. However, now it seems to just do one action and then stop, is there other configuration / crontab or such that I am missing?
